cmd/helm.go

```diff
@@ -587,6 +622,25 @@ func grantHelmAccess(cliContext *cli.Context) error {
 	return helm.GrantAccess(kubectlOptions, tlsOptions, tillerNamespace, rbacGroups, rbacUsers, serviceAccounts)
 }
 
+// revokeHelmAccess is the action function for the helm revoke command.
+func revokeHelmAccess(cliContext *cli.Context) error {
+	tillerNamespace, err := entrypoint.StringFlagRequiredE(cliContext, tillerNamespaceFlag.Name)
+	if err != nil {
+		return err
+	}
+	kubectlOptions, err := parseKubectlOptions(cliContext)
+	if err != nil {
+		return err
+	}
+	rbacGroups := cliContext.StringSlice(grantedRbacGroupsFlag.Name)
+	rbacUsers := cliContext.StringSlice(grantedRbacUsersFlag.Name)
+	serviceAccounts := cliContext.StringSlice(grantedServiceAccountsFlag.Name)
+	if len(rbacGroups) == 0 && len(rbacUsers) == 0 && len(serviceAccounts) == 0 {
+		return entrypoint.NewRequiredArgsError("At least one --rbac-group, --rbac-user, or --rbac-service-account is required")
```

Comment:
Consider using %s for the arg names and replacing them with `grantedRbacGroupsFlag.Name`, `grantedRbacUsersFlag.Name`, etc.

Reply:
Yep, made this change!